glib/src/auto/functions.rs

```diff
@@ -480,6 +622,16 @@ pub fn mkdir_with_parents(pathname: impl AsRef<std::path::Path>, mode: i32) -> i
     unsafe { ffi::g_mkdir_with_parents(pathname.as_ref().to_glib_none().0, mode) }
 }
 
+#[doc(alias = "g_mkdtemp")]
+pub fn mkdtemp(tmpl: impl AsRef<std::path::Path>) -> Option<std::path::PathBuf> {
```

Comment:
This and `mkdtemp_full` needs to stay manual because of the comment that was in the manual code.